Resources/public/css/views/linkmanager.css

```diff
@@ -0,0 +1,37 @@
+.ez-linkmanager-link-list {
+    min-width: 1024px;
+}
+
+.ez-linkmanager-filters {
+    clear: both;
+}
+
+.ez-linkmanager-filter-search {
+    float: left;
+    width: 50%;
+}
+
+.ez-linkmanager-filter-status {
```

Comment:
You are using `float`. Do you use any kind of clearfix as well?

Reply:
I've added `clear: both` for `.ez-linkmanager-link-list` and I hope it's OK

Reply:
👍